src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -1986,7 +1986,7 @@ namespace OpenMS
       // set precursor information
       vector<Precursor> precursors;
       Precursor precursor;
-      precursor.setMZ(aa_sequence.getMonoWeight());
+      precursor.setMZ(aa_sequence.getMZ(charge));
```

Comment:
no the same, is it?
Was it a bug before?

Reply:
it was not used but getMZ(charge) is the correct precursor m/z so it was kind of a bug